src/Directory/Biobanks.Web/Controllers/BiobankController.cs

```diff
@@ -1004,6 +1004,32 @@ public async Task<ActionResult> GetAssociatedDataTypeViewsAjax(string id)
 
         }
 
+        private async Task<Boolean> linkedAssociatedDataIsValid(
```

Comment:
- C# Methods should be `PascalCase`
- boolean "questions" should start with the question bit e.g. `IsThisValid` not `ThisIsValid`

```suggestion
        private async Task<Boolean> IsLinkedAssociatedDataValid(
```